chapters/5-linux-git.md

```diff
@@ -0,0 +1,73 @@
+# Linux and Git Commands
+
+The following sections list of the more commonly used Linux and Git
+commands.
+
+You will find them useful for the lesson on Version Control Systems.
+
+Most of these commands have tons of options you will seldom use.
+We'll cover some of the more frequently used ones here, but you can always
+find out more by checking the command's documentation.
+
+In Linux (and OS X), you can look up the documentation for a command with
+`man` (manual).
+For example, `man ls` opens the manual page for the `ls` command.
+
+`man man` displays documentation for the `man` command itself!
+
+## Some Conventions
+
+In this cheatsheet, `<something>` indicates that `something` is a variable.
+You should replace it with the thing you want the command to act on.
+
+`<something ...>` means that the command can accept multiple arguments.
```

Comment:
Would `<something> <...>` be clearer?
Otherwise `<directory name>` might give the impression that we are passing two arguments to cd.